app/src/organisms/LabwarePositionCheck/LabwarePositionCheckComponent.tsx

```diff
@@ -261,6 +261,8 @@ export const LabwarePositionCheckComponent = (
   const currentStep = LPCSteps?.[currentStepIndex]
   if (currentStep == null) return null
 
+  const protocolHasModules: boolean = protocolData.modules.length > 0
```

Comment:
```suggestion
  const protocolHasModules = protocolData.modules.length > 0
```
Nit, with TypeScript it's best practice to let TS do type inference when possible. 

Your variable name is great - it conveys that this ought to be a boolean, which further helps!

Reply:
makes sense— I will refer to this going forward